react/ContextHeader/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button } from '../Button'
+import Icon from '../Icon'
+import styles from './styles.styl'
+import palette from '../../stylus/settings/palette.json'
+
+const ContextHeader = ({ icon, title, text, onClose }) => {
+  return (
+    <header className={styles['context-header']}>
```

Comment:
`<header>` element is most likely to wrap headings, like titles and subtitles. Not quite sure this component can be related to a header per se. 
I would use a simple `<div>` for this one. It's not because it's the first element of the container that is should be a `<header>`, actually `<header>` is not a required element.